Source/WebKit/WebProcess/Plugins/PDF/PDFPlugin.mm

```diff
@@ -647,6 +647,9 @@ static void getAllScriptsInPDFDocument(CGPDFDocumentRef pdfDocument, Vector<Reta
 #endif
     , m_identifier(PDFPluginIdentifier::generate())
 {
+    [[NSUserDefaults standardUserDefaults] setBool:NO forKey:@"PDFKit2_UseIOSurfaceForTiles"];
```

Comment:
Let's add a comment referencing a radar for converting this to use SPI.

Reply:
Fixed.